plugins/inventory/icinga2.py

```diff
@@ -63,6 +65,20 @@
 host_filter: \"linux-servers\" in host.groups
 validate_certs: false
 inventory_attr: name
+groups:
+  # simple name matching
+  webservers: inventory_hostname.startswith('web')
+
+  # using icinga2 template
+  databaseservers: "'db-template' in (icinga2.templates|list)"
+
+compose:
+  # set all icinga2 attributes to a host variable 'icinga2_attrs'
+  icinga2_attrs: icinga2_attributes
+
+  # set 'ansible_user' and 'ansible_port' from icinga2 host vars
+  ansible_user: icinga2_attributes.vars.ansible_user
+  ansible_port: icinga2_attributes.vars.ansible_port
```

Comment:
Maybe also show that the `default` filter can be useful here:
```suggestion
  ansible_port: icinga2_attributes.vars.ansible_port | default(22)
```
(Not everyone knows of that filter.)